overview.md

```diff
@@ -43,130 +43,173 @@ steps:
+| `products` | <p>(Optional) Products to install in addition to MATLAB, specified as a list of product names separated by spaces. You can specify `products` to install most MathWorks products and support packages. For example, `products: Deep_Learning_Toolbox` installs Deep Learning Toolbox&trade; in addition to MATLAB.</p><p>The task uses [MATLAB Package Manager](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md) (`mpm`) to install products. For a list of supported products and their correctly formatted names, see [Product Installation Options](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md#product-installation-options).</p><p>**Example**: `products: Simulink`</br>**Example:** `products: Simulink Deep_Learning_Toolbox`</p>
 
-Argument                  | Description    
-------------------------- | --------------- 
-`release`                 | (Optional) MATLAB release to install. You can specify R2021a or a later release. By default, the value of `release` is `latest`. If you do not specify `release`, the task installs the latest release of MATLAB.<br/>**Example:** `release: R2023b`<br/>**Example:** `release: latest`
-`products`                | (Optional) Products to install in addition to MATLAB, specified as a list of product names separated by spaces. You can specify `products` to install most MathWorks&reg; products and support packages. For example, `products: Deep_Learning_Toolbox` sets up Deep Learning Toolbox&trade; in addition to MATLAB. The task uses [MATLAB Package Manager](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md) (`mpm`) to install products. For a list of supported products and their correctly formatted names, see [Product Installation Options](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md#product-installation-options).<br/>**Example:** `products: Simulink`<br/>`products: Simulink Deep_Learning_Toolbox`
+#### Licensing
+Product licensing for your pipeline depends on your project type as well as the type of products to install:
+
+- Public project — If your pipeline does not include transformation products, such as MATLAB Coder and MATLAB Compiler, then the task automatically licenses any products that you install. If your pipeline includes transformation products, you can request a MATLAB batch licensing [token](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/alternates/non-interactive/MATLAB-BATCH.md#matlab-batch-licensing-token) by contacting MathWorks at [batch-tokens@mathworks.com](mailto:batch-tokens@mathworks.com).
+- Private project — The task does not automatically license any products for you. You can request a batch licensing token by contacting MathWorks.
```

Comment:
The `InstallMATLAB` task is not the one licensing MATLAB. It's the `Run*` tasks that do that. I would consider changing "The task does not automatically" to something like "MATLAB is not automatically licensed"

Reply:
Good point.

Using "The extension does not automatically license"